src/hooks/useVaultRouter.ts

```diff
@@ -26,6 +26,7 @@ export function useVaultRouter(contracts?: Contracts) {
 
     const tokenAddress = getTokenAddress(vault);
     const token = contracts.getERC20(tokenAddress);
+    const symbol = await token.symbol();
     const myAddress = await contracts.getMyAddress();
     const allowance = await token.allowance(
```

Comment:
do you think it might be better to Promise.all this sequence of async calls, or would it still not be worth it?